nitro-protocol/contracts/test/TESTStrictTurnTaking.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+pragma experimental ABIEncoderV2;
+
+import '../interfaces/INitroTypes.sol';
+import {StrictTurnTaking} from '../libraries/signature-logic/StrictTurnTaking.sol';
+
+/**
+ * @dev This contract uses the StrictTurnTaking library to make it more easily unit-tested. It exposes public or external functions call into internal functions. It should not be deployed in a production environment.
```

Comment:
```suggestion
 * @dev This contract uses the StrictTurnTaking library to make it more easily unit-tested. It exposes public or external functions which call into internal functions. It should not be deployed in a production environment.
```

and similarly for the other `TEST*.sol` files.

Reply:
d3aeefb544fa6f65d978f5b14e4c0b01ec4568b0